### DISCUSSION TITLE
How to custom day note name created by calendar widget?

### DISCUSSION THREAD

**Discussion by ghost** [TriliumNext/Trilium#2017] [Q&A]
When day note is created by clicking calendar widget, note name have the form 'day of month - day of week'. But i want custom the name. 1 way i can think of is using #datePattern={isoDate}, but this will make all the note name is today date, not the correspond day. So, how can i custom day note name created by calendar widget?

**Comment by XiaofengdiZhu** [comment]
[https://github.com/zadam/trilium/search?q=getDateNoteTitle](https://github.com/zadam/trilium/search?q=getDateNoteTitle)

**Reply by ghost** [reply]
I want it to be some code note or a label in root note, not changing the source code of trilium itself.
